config/database.yml

```diff
@@ -24,6 +24,9 @@ default: &default
 development:
   <<: *default
   database: hello_world_back_end_development
+  username: nakitto
+  password: Mangoes2012
+  port: 5433
 
```

Comment:
## [OPTIONAL]

- [ ] Exposing your database credential is very dangerous since this is the base of the application. And to prevent this you may use environment variables to store this secret information check here for [more info](http://railsapps.github.io/rails-environment-variables.html).